src/dmqproto/client/DmqClient.d

```diff
@@ -361,6 +396,41 @@ public class SchedulingDmqClient : ExtensibleDmqClient!(RequestScheduler)
             new RequestScheduler(epoll, max_events),
             conn_notifier, conn_limit, queue_size, fiber_stack_size);
     }
+
+
+    /***************************************************************************
+
+        Constructor with support for the neo and legacy protocols. Automatically
+        calls addNodes() with the node definition files specified in the legacy
+        and neo Config instances.
+
+        Params:
+            epoll = EpollSelectDispatcher instance to use
+            config = swarm.client.model.IClient.Config instance. (The Config
+                class is designed to be read from an application's config.ini
+                file via ocean.util.config.ConfigFiller.)
+            neo_config = swarm.neo.client.mixins.ClientCore.Config instance.
+                (The Config class is designed to be read from an application's
+                config.ini file via ocean.util.config.ConfigFiller.)
+            conn_notifier = delegate which is called when a connection attempt
+                succeeds or fails (including when a connection is
+                re-established). Of type:
+                void delegate ( IPAddress node_address, Exception e )
+            fiber_stack_size = size (in bytes) of stack of individual connection
+                fibers
+            max_events = limit on the number of events which can be managed
+                by the scheduler at one time. (0 = no limit)
+
+    ***************************************************************************/
+
+    public this ( EpollSelectDispatcher epoll, IClient.Config config,
```

Comment:
This should accept a `SchedulingDmqClient.Config`. See https://github.com/sociomantic-tsunami/dhtproto/pull/57/files. (Spotted by Joe.)